### ISSUE TITLE
web-mode template issues for django engine

### ISSUE THREAD

**Issue by synic** [syl20bnr/spacemacs#4499]
If you type `{{var`, smartparens enters `}}` correctly.

If you type `{{ var`, you will get `}}}`.

Not sure if this is a spacemacs issue, or a web-mode issue, or smart-parens or what.  It happens in other engines too.

#### System Info
- OS: darwin
- Emacs: 24.5.7
- Spacemacs: 0.105.4
- Spacemacs branch: develop (rev. c34c9d0)
- Distribution: spacemacs
- Layers:
```elisp
(unscroll auto-completion xkcd emacs-lisp git dash yaml github javascript html markdown extra-langs python ruby org osx python django syntax-checking spell-checking version-control lua colors spacemacs-layouts vimscript themes-megapack
          (c-c++ :variables c-c++-enable-clang-support t)
          (shell :variables shell-default-height 30 shell-default-position 'bottom))
```

**Comment by TheBB** [comment]
It's probably a smartparens issue. Spacemacs has disabled web-mode's built-in automatic pairing and uses smartparens instead. If I revert that, it works fine for me:

```lisp
  (add-hook 'web-mode-hook 'spacemacs/toggle-smartparens-off 'append)
  (setq web-mode-enable-auto-pairing t)
```

I'm not sure why the original decision was made but I'm leaning towards scrapping smartparens in web-mode because this is incredibly annoying.

**Comment by synic** [comment]
I wonder if it's so that `SPC t p` works correctly in web-mode.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
